src/main/java/mate/academy/dao/impl/CinemaHallDaoImpl.java

```diff
@@ -40,19 +40,67 @@ public Optional<CinemaHall> get(Long id) {
         try (Session session = HibernateUtil.getSessionFactory().openSession()) {
             return Optional.ofNullable(session.get(CinemaHall.class, id));
         } catch (Exception e) {
-            throw new DataProcessingException("Can't get a cinema hall by id: " + id, e);
+            throw new DataProcessingException("Can't get a cinemaHall by id: " + id, e);
         }
     }
 
     @Override
     public List<CinemaHall> getAll() {
         try (Session session = HibernateUtil.getSessionFactory().openSession()) {
-            CriteriaQuery<CinemaHall> criteriaQuery = session.getCriteriaBuilder()
-                    .createQuery(CinemaHall.class);
-            criteriaQuery.from(CinemaHall.class);
-            return session.createQuery(criteriaQuery).getResultList();
+            return session.createQuery("from CinemaHall").list();
+        } catch (Exception e) {
+            throw new DataProcessingException("Can't get cinemaHalls", e);
+        }
+    }
+
+    @Override
+    public boolean update(CinemaHall cinemaHall) {
```

Comment:
Consider changing the return type of the `update` method to `void` or `CinemaHall`. The current implementation returns a boolean based on equality check, which might not be reliable or meaningful.